platforms/python/django-profiling/requirements-sentry.txt

```diff
@@ -1 +1 @@
-sentry-sdk==1.3.0
+sentry-sdk @ https://github.com/getsentry/sentry-python/archive/e3d245a293802ae0c721ea95889b905114004436.zip
```

Comment:
This will get released soon, so we can pin it to an SDK version once that happens.